operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -38,26 +39,13 @@ local filename = 'mimir-reads.json';
          showTitle: false,
        })
       .addPanel(
+        local addRuleEvalRate(q) = {
+          local ruleEvalRate = ' + sum(rate(cortex_prometheus_rule_evaluations_total{' + $.jobMatcher($._config.job_names.ruler) + '}[$__rate_interval]))',
```

Comment:
note to reviewers: `sum(x or y)` is the same as `sum(x)+sum(y)` because x and y have different metric names.

Reply:
[That is not true.](https://promql.info/?b64=IyBEaWZmZXJlbnQgbWV0cmljIG5hbWVzLCBidXQgc2FtZSAicmVndWxhciIgbGFiZWxzLg0KbG9hZCA1bQ0KICBmb297aW5zdGFuY2U9ImFiYyJ9IDINCiAgYmFye2luc3RhbmNlPSJhYmMifSAxDQoNCiMgRXZlbiB0aG91Z2ggdGhlIG1ldHJpYyBuYW1lIGRpZmZlcnMsIGBvcmAgb25seSByZXR1cm5zIG9uZSBzZXJpZXMsIGJlY2F1c2UgbWV0cmljIG5hbWUgaXMgaWdub3JlZC4NCmV2YWwgaW5zdGFudCBhdCA1bSBmb28gb3IgYmFyDQogIGZvb3tpbnN0YW5jZT0iYWJjIn0gMg0KDQojIFNvIHRoZSBzdW0gb2YgYGZvbyBvciBiYXJgIGlzIGp1c3Qgc3VtIG9mIGBmb29gDQpldmFsIGluc3RhbnQgYXQgNW0gc3VtKGZvbyBvciBiYXIpDQogIHt9IDINCg0KIyBXaGljaCBpcyBkaWZmZXJlbnQgZnJvbSB0aGUgc3VtIG9mIHN1bXMuDQpldmFsIGluc3RhbnQgYXQgNW0gc3VtKGZvbykgKyBzdW0oYmFyKQ0KICB7fSAz)

Reply:
On the other hand it means that the new query is actually more truthful since it's not going drop data in case labels excluding `__name__` line up.